### DISCUSSION TITLE
What about the icons?

### DISCUSSION THREAD

**Discussion by tkoop** [ueberdosis/tiptap#832] [General Questions]
I'm new to Vue, so I'm trying to use tiptap by pasting in the example code.  But it's complaining about icons.  What icons are you using?  And how can I install them?

Perhaps there could be a note about installing the icon pack you are using somewhere?

Thanks.

**Comment by hanspagel** [comment]
Hi tkoop! Thanks for asking! 

Tiptap is renderless, that means we don’t want to provide the look of the editor. The icons are used for the examples only. Feel free to add your custom <Icon> component with the icon set you like. I think we are using feather icons.

That said, we’re working to improve this with the next version of tiptap which will probably come with a starter theme.

Hope that helps!
